### PR TITLE
add coverage config file

### PR DESCRIPTION
**Pull request by jpchen**
generating accurate coverage reports. all the ignores are:
- warnings
- errors (this is standard in pytest-cov config files)
- backward implementations (this is tested but the control flow goes to pytorch then back to pyro, which cant be checked by static analysis)
- main methods (we actually test this, though copied from the pytest-cov example config)

testing on CI

### CODE REVIEW

**Comment by martinjankowiak** [comment]
"backward implementations (this is tested but the control flow goes to pytorch then back to pyro, which cant be checked by static analysis)"

wow that sounds an awful lot like something i said 🙄

File: .codecov.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
 ignore:
   - "pyro/docutil.py"
   - "pyro/logger.py"
+
+coverage:
+  range: 60..95
```
Comment (neerajprad) [inline .codecov.yml]: What does this range represent?

Comment (jpchen) [reply .codecov.yml]: how green that badge is :D

Comment (neerajprad) [reply .codecov.yml]: Hahaha.. 👍

Comment (jpchen) [reply .codecov.yml]: i know what youre going to say but this is vital stuff :P

Comment (neerajprad) [reply .codecov.yml]: To be honest..that light green badge was irking me a bit too.

**Pull request merged**